modules/Quiz/resources/IVQModule.js

```diff
@@ -360,22 +362,24 @@
              * @param {*} openQuestionText 
              */
             submitAnswer:function(questionNr,selectedAnswer,openQuestionText){
-                var _this = this,isAnswered;
+                var _this = this;
+                var defer = $.Deferred();
                 $.cpObject.cpArray[questionNr].selectedAnswer = selectedAnswer;
-                if ($.cpObject.cpArray[questionNr].isAnswerd) {
-                    isAnswered = true;
-                }
-                else{
-                    isAnswered = false;
-                    $.cpObject.cpArray[questionNr].isAnswerd = true;
-                }
-                _this.KIVQApi.addAnswer(isAnswered,_this.i2q(selectedAnswer),_this.kQuizUserEntryId,questionNr,function(data){
+                $.cpObject.cpArray[questionNr].submitRequested = true;
+                var submitCallTimeout = setTimeout(function(){
+                    defer.reject();
+                }, _this.quizPlugin.getConfig("apiRequestTimeout") || 5000)
```

Comment:
better explicitly add a new configuration and set the default there instead of checking this with every time we submit answer